test/extensions/stats_sinks/hystrix/hystrix_test.cc

```diff
@@ -428,6 +430,39 @@ TEST_F(HystrixSinkTest, AddAndRemoveClusters) {
   // Check that old values of test_cluster2 were deleted.
   validateResults(cluster_message_map[cluster2_name_], 0, 0, 0, 0, 0, window_size_);
 }
+
+TEST_F(HystrixSinkTest, HystrixEventStreamHandler) {
+  InSequence s;
+  Buffer::OwnedImpl buffer = createClusterAndCallbacks();
+  // Register callback to sink.
+  sink_->registerConnection(&callbacks_);
+
+  // This value doesn't matter in handlerHystrixEventStream
+  absl::string_view path_and_query;
+
+  Http::HeaderMapImpl response_headers;
+  Stats::IsolatedStoreImpl listener_scope_;
+
+  NiceMock<Envoy::Server::Configuration::MockAdminStream> admin_stream_mock;
+
+  ON_CALL(admin_stream_mock, setEndStreamOnComplete(_)).WillByDefault(Return());
+  ON_CALL(admin_stream_mock, addOnDestroyCallback(_)).WillByDefault(Return());
+  ON_CALL(admin_stream_mock, getDecoderFilterCallbacks()).WillByDefault(ReturnRef(callbacks_));
+
+  ASSERT_EQ(
+      sink_->handlerHystrixEventStream(path_and_query, response_headers, buffer, admin_stream_mock),
+      Http::Code::OK);
+
+  // Check that response_headers has been set correctly
+  EXPECT_EQ(std::string(response_headers.ContentType()->value().getStringView()),
+            "text/event-stream");
+  EXPECT_EQ(std::string(response_headers.CacheControl()->value().getStringView()), "no-cache");
```

Comment:
Do you need to explicitly promote to `std::string`? Can't equality on `string_view` handle this?